### PR TITLE
Remove play dependence.

### PR DESCRIPTION
**Pull request by rabbah**
This removes the play dependence introduced in https://github.com/apache/incubator-openwhisk/commit/55e693a4d91ea18aed0a01740db0ac2b9f56cddb.

@csantanapr you may want to try this as part of your docker mac efforts to make sure the action container unit tests can run with this change.

@style95 want to try this?

### CODE REVIEW

**Comment by rabbah** [comment]
@csantanapr you may want to try this as part of your docker mac efforts.

**Comment by style95** [comment]
Yes sure.
I will try.

**Comment by style95** [comment]
Some tests are failing, I am figuring out the reason.

**Comment by style95** [comment]
I am facing this error in many test cases.
```
whisk.core.WhiskConfigTests > initializationError FAILED
    java.lang.OutOfMemoryError: unable to create new native thread
        at java.lang.Thread.start0(Native Method)
        at java.lang.Thread.start(Thread.java:714)
        at akka.actor.LightArrayRevolverScheduler.<init>(LightArrayRevolverScheduler.scala:299)
        at sun.reflect.GeneratedConstructorAccessor63.newInstance(Unknown Source)
        at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
        at java.lang.reflect.Constructor.newInstance(Constructor.java:423)
        at akka.actor.ReflectiveDynamicAccess$$anonfun$createInstanceFor$2.apply(ReflectiveDynamicAccess.scala:32)
        at scala.util.Try$.apply(Try.scala:192)
        at akka.actor.ReflectiveDynamicAccess.createInstanceFor(ReflectiveDynamicAccess.scala:27)
        at akka.actor.ReflectiveDynamicAccess$$anonfun$createInstanceFor$3.apply(ReflectiveDynamicAccess.scala:38)
        at akka.actor.ReflectiveDynamicAccess$$anonfun$createInstanceFor$3.apply(ReflectiveDynamicAccess.scala:38)
        at scala.util.Success.flatMap(Try.scala:231)
        at akka.actor.ReflectiveDynamicAccess.createInstanceFor(ReflectiveDynamicAccess.scala:38)
        at akka.actor.ActorSystemImpl.createScheduler(ActorSystem.scala:859)
        at akka.actor.ActorSystemImpl.<init>(ActorSystem.scala:761)
        at akka.actor.ActorSystem$.apply(ActorSystem.scala:245)
        at akka.actor.ActorSystem$.apply(ActorSystem.scala:288)
        at akka.actor.ActorSystem$.apply(ActorSystem.scala:233)
        at akka.actor.ActorSystem$.apply(ActorSystem.scala:224)
        at common.WskActorSystem$class.$init$(WskActorSystem.scala:39)
        at whisk.core.WhiskConfigTests.<init>(WhiskConfigTests.scala:38)
```

It seems it can not create more thread.
I am looking for the way to configure JVM for this.

**Comment by rabbah** [comment]
hmm do you think the jvm issue is related to this change?

File: core/invoker/src/main/scala/whisk/core/container/WhiskContainer.scala [hunk 1]
```diff
@@ -120,7 +120,7 @@ class WhiskContainer(
      * Tear down the container and retrieve the logs.
      */
     def teardown()(implicit transid: TransactionId): String = {
-        connection.foreach(_.close)
+        connection.foreach(_.close())
```
Comment (rabbah) [inline core/invoker/src/main/scala/whisk/core/container/WhiskContainer.scala]: @markusthoemmes i think this was missing from the new pool, added it for good measure.

**Comment by style95** [comment]
@rabbah 
I also don`t think so.
Surprisingly, that error has came up that I have not seen that before.

Are you able to run all tests?

**Comment by rabbah** [comment]
do you get the error if you're not on this branch?

**Comment by rabbah** [comment]
pg3/812 🔵

**Comment by rabbah** [comment]
@csantanapr i assigned this to you because you're running docker for mac - i thought.
this change works for docker-machine and removed the play dependence which was added by @style95 as part his pr on docker for mac. He recently confirmed that he can still run these tests without play.

**Pull request merged**